generated_bindata.go

```diff
@@ -89,7 +89,7 @@ func dataBuild() (*asset, error) {
6\xf1\xa1\xc3\xc4\xb6\x80\x31\x97\x66\x77\x1a\xc0\x0c\x35\xd1\x17\x0a\x77\x72\x43\x76\x75\x0d\xea\xc5\x02\x2a\x31\x8f\x6b\x47\x44\x12\xce\xfd\x2a\xdc\xc6\xb7\xb0\x10\x6f\x82\x9e\x78\x23\xf0\x46\x68\x6f\x8f\x6d\x0f\x4d\xad\xaf\x15\xaa\x58\x0f\x23\x76\x97\x8f\xfa\xed\xc8\x28\x5c\xcb\x2c\x11\x1b\x6d\x9e\x69\x09\x05\x78\x1a\xd7\x52\xf3\x32\x22\xe2\x1e\xe1\x2b\x83\x21\x37\x3a\x91\xf0\xeb\xbc\x0d\xae\x25\x4f\x24\x03\x71\xad\x60\x6e\x47\x6c\x8a\xf3\x10\x7a\x3c\x5a\x58\x89\x6f\x12\x5a\x5c\xcb\x96\x48\xf8\x38\xd5\x94\x48\xf2\x44\x32\x12\xd7\x32\xb7\x23\xb6\x35\xb2\x60\x01\x29\xe0\xa2\x12\xb5\xdb\xae\x4f\x24\x36\x03\xac\x80\xaa\x52\x02\xd7\x92\x27\x12\x1a\xd7\x1a\xac\x38\x3f\xf9\xbe\x77\x63\xda\xe4\xfb\x1a\x2e\xc2\xe4\xfb\xde\x96\xad\x93\xef\x3b\xf9\xbe\x5f\x7f\xd3\xa3\x6e\x8c\x5c\x40\x6f\x9e\xce\x22\x88\x51\xc0\x67\x11\x9e\x0b\xd5\xaf\xdb\x0d\x9d\x48\xd4\xab\x2f\x59\x74\xf4\xd1\x24\x96\xf3\xbe\x52\x8b\x43\x6f\xe2\x1b\x93\xc6\x51\xb5\xf0\x92\x2e\x5f\x7e\xda\x20\x2d\x91\x1e\x60\x53\x7e\xea\x07\x23\x3c\xf8\xda\xd6\x20\x36\x79\x3e\x3d\x3f\x27\xc8\x17\x96\x07\x52\xb5\x92\xb9\xa4\xe0\xb9\x50\xfd\x4a\xc4\x82\x6e\xea\x81\x6e\x8b\x0d\x9a\xcd\x60\x40\xa0\x98\xde\x54\x48\x62\x79\xbe\x48\x4f\x46\x46\x1b\x0c\xc0\xb8\xc9\xe6\x10\xde\xf8\xbe\x0f\x82\x59\x3a\x11\xfd\x18\x0d\x9a\x21\x12\x39\x84\xd7\x9a\xed\x83\x32\x5a\x31\xc7\x14\xe4\x39\x64\x9c\x36\xf4\x70\x1b\x34\x19\x7c\xd6\x09\xba\xda\x1c\x52\x86\xc9\x84\xab\x81\xcd\x7b\x4a\x16\xdc\x70\x7d\x5c\x0e\x81\x71\x93\x6f\x4b\x8d\xf6\xdc\xa6\xb6\x54\x8f\x4f\x20\x58\x9c\x6a\x4b\x8d\x2c\x2d\x2d\xcf\x22\x6c\x62\x8c\xfb\x9e\x30\xf0\x35\x58\x0c\x60\x8c\x99\x6f\x45\x4c\x43\x11\x3c\x28\x27\xa7\xeb\xea\x0b\x63\x69\x45\xa0\x78\xd0\xd1\xad\xc8\x2a\xf7\xe7\x8a\xe3\xa6\xeb\x66\xba\x9e\x80\x04\x75\x20\xc3\x53\x0f\x69\x19\x5a\x11\xde\x04\xa5\xdf\x8a\x68\xeb\xcf\x18\xc4\x0b\x44\xa7\xeb\x39\xac\x15\x31\x57\x14\x3c\x09\xaa\x7f\x70\x16\x48\xab\xe8\x21\x2d\x4b\x2b\x62\x4b\x23\x3c\xbb\xa7\xc9\x8a\x72\xc1\xdf\x40\x0e\x45\x06\x5b\x0c\xc8\x86\x9b\xfd\x0a\xdc\x66\x31\x20\xaa\x72\x1a\x36\xa4\xda\xb0\xd1\xc2\xd0\xca\xc6\x1c\xa2\x1e\xa1\xc9\x6f\x4b\x42\xfe\x83\x97\x60\x68\xd9\x95\xc2\x6d\x39\xc4\xa6\xbf\xa7\x77\x64\xb6\x90\x3b\x8d\xfa\x7b\x7a\x86\x96\x82\xdc\x09\x26\x64\x97\x60\x68\x99\xcb\x89\x68\xcb\x21\x08\x05\xa3\x19\x5a\x7a\xf3\x4b\x5e\xc8\x5c\x0e\xed\x1d\xce\xd0\x92\xab\x72\x7a\xce\xed\x6c\xe8\x60\x1d\xd5\x68\x82\x3e\xc4\xa6\xe1\x8a\xb0\x05\xba\x0f\x51\xdf\x16\xb8\x42\x47\xf5\x21\x26\x53\xa3\xc8\xb7\xc8\x4d\x1f\x62\xdc\x53\xbd\x70\x0e\xb1\x2e\x1d\x9a\x37\xcd\x0c\x36\x89\x98\xc6\x43\x44\x01\xc3\xca\x32\xfb\x5b\xfa\xe5\xb4\x1a\xce\x52\x68\xb8\x96\x81\x39\xc4\xbc\x74\x88\x58\x6d\x5c\x1f\x62\x18\x94\x06\xa4\xe2\x4f\xe5\x10\xbd\xf1\x08\x64\x02\x31\x7d\x88\x1a\xce\x92\xbb\xe8\xc6\xae\xe7\xf2\x77\x37\x62\x8b\x81\x72\x9b\x6f\x9e\x99\xc5\xc0\xdf\x54\x1e\x03\xa1\x3e\x92\xc9\xc0\xfa\xf5\x1f\x1f\xdf\x64\xe0\xef\xdf\x7f\xfe\xcb\x9b\x57\x57\xea\x35\x50\xa0\x3b\x32\x17\x87\x0a\x4f\xa7\xea\x3e\x66\x41\x49\x0b\xbf\x12\xb2\x7f\xb1\xcc\x3b\xb1\x34\x67\x61\xab\x4e\xb
```

Comment:
What changed here? Let's keep this file out of this PR, or maybe submit it independently in another PR.